src/graph/resolvers/campaign.js

```diff
@@ -148,16 +159,20 @@ module.exports = {
       const campaign = await Campaign.findById(campaignId);
       if (!campaign) throw new Error(`Unable to set campaign URL: no campaign found for '${campaignId}'`);
       campaign.url = url;
-      return campaign.save();
+      await campaign.save();
+      await updateNotifications(campaignId);
```

Comment:
Does `updateNotifications` need to be called on _every_ mutation?

Reply:
IMO yes, as any campaign field could be used within the email message itself, and changes to criteria would necessitate changes to delivery